library/general/src/lib/installation/auto_client.rb

```diff
@@ -99,8 +100,9 @@ def import(_profile)
     #
     # The profile is a Hash or an Array according to the configuration item
     # `X-SuSE-YaST-AutoInstDataType`
+    # @param _params [Hash<String,Object>] Additional parameters
```

Comment:
I would document such parameters with `@option` yardoc tag

Reply:
Thanks for your review. This comment takes me to another question: would it be better to define this method as:

```ruby
export(target: target)
```

Using a `_params` allows us to extend it easier in the future (without having to touch this code anymore). But an explicit declaration could make the code more clear (and easier to read). After all, I do not expect to start adding arguments to `export` every month. ;-) And Ruby already offers quite some control about how to detect which arguments are supported.

Any though about this?

Reply:
keywords based looks better for me. And having symbol as param also looks fine for me. It will also provide one point where default is defined if we convert all auto clients to objects.